### PR TITLE
complete 3.5/api

### PR DESCRIPTION
**Pull request by dave-hickman**

### CODE REVIEW

File: __tests__/endpoints.test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+afterAll(() => {
+    return db.end
+})
+
+describe('GET /api', () => {
+    it('should return a JSON object', () => {
+        return request(app).get('/api')
+        .expect(200)
+        .then((response) => {
+            console.log(response.text)
```
Comment (JVal-NC) [inline __tests__/endpoints.test.js]: No console logs in PRs please

File: __tests__/endpoints.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            expect(JSON.parse(response.text)).toBeInstanceOf(Object)
+        })
+        
+    });
+    it('should contain GET /api key, api/topics and api/articles keys', () => {
```
Comment (JVal-NC) [inline __tests__/endpoints.test.js]: Not bad tests but what we can do is simply require the json file at the top of our test file and test our endpoints are returning that on the body.

**Review by JVal-NC — COMMENTED** [review]
Dave, good start but another PR please once you've updated for the comments.

**Review by pippaamy — APPROVED** [review]
fab changes! go ahead and merge :)

**Pull request merged**